_posts/2018-06-13-get-ready-to-rebble.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Getting Ready for Rebble"
+date:   2018-06-13 12:00:00
+# categories: community
+---
+
+Hi again! I'm [Katharine Berry](https://twitter.com/KatharineBerry), back
+with a brief update about the Rebble Web Services. As you have all probably noticed,
+we've been a bit quiet since my [last post](/2018/02/15/rebble-web-services.html).
+Unfortunately, life happened for all of us, as it tends to. But we're back on
```

Comment:
I adore pretty much the whole blog post except this once sentence, for two reasons: 1) this is a community project, that we all do for our own enjoyment or other personal reasons, and nobody "owes" anyone anything, so no excuses/explanations are required IMO; I'd go with something more like:

> Given our limited resources, we often have to choose between making progress and communicating progress made.   We're finally taking a moment for the latter, but before we go any further, we have something important for you to do!

2) it creates room for doubt i.e. "what happens to the project next time 'life happens'?" vs. the more positive assumption that when we're not blogging it's because we're working on stuff to blog about! ❤️

Reply:
I think I disagree on this — the web services made near-zero progress for four months, and to say we were heads down on it is untrue. Anyone can check the github org and see that (and they have).

As for the doubt — well, it's a legitimate one to have. I _hope_ we'll make it through, but that's the best I can realistically offer.

Reply:
@Katharine huh - I certainly wasn't intending to suggest that we be anything but honest; were you referring purely to RWS with your statement?  I read "all of us" as implying that nothing was happening anywhere, which is [demonstrably](https://github.com/ginge/FreeRTOS-Pebble/commits/master) [false](https://github.com/pebble-dev/neographics/commits/master); could we perhaps incorporate language making it clear that firmware continues to march forward regardless of delays on the RWS front? 😅 ❤️

Reply:
I've changed the language slightly; perhaps the new language is more to your liking?